README.md

```diff
@@ -154,6 +156,15 @@ sudo cp target/release/alacritty /usr/local/bin # or anywhere else in $PATH
 cp Alacritty.desktop ~/.local/share/applications
 ```
 
+#### OS X
+
+To install the desktop entry for Alacritty, run
+
+```sh
+make app
```

Comment:
Yes and thanks for adding this to README!!

IIRC, the Makefile targets all require that you've already run `cargo build --release`, right?